dataduct/steps/load_reload_pk.py

```diff
@@ -0,0 +1,195 @@
+               analyze_table, non_transactional):
+        source_table = parse_path(source)
+        destination_table = parse_path(destination)
+
+        source_relation = Table(SqlScript(filename=source_table))
+        destination_relation = Table(SqlScript(filename=destination_table))
+
+        # Reload specific config
+        enforce_primary_key = True
+        delete_existing = True
+        sql_script = destination_relation.upsert_script(
+            source_relation, enforce_primary_key, delete_existing)
+
+        update_script = SqlScript(sql_script.sql())
+        script_arguments = [
+            '--table_definition=%s' % destination_relation.sql(),
+            '--sql=%s' % update_script.sql()
+        ]
+
+        if analyze_table:
+            script_arguments.append('--analyze')
+
+        if non_transactional:
+            script_arguments.append('--non_transactional')
+
+        steps_path = os.path.abspath(os.path.dirname(__file__))
+        script = os.path.join(steps_path, const.SQL_RUNNER_SCRIPT_PATH)
+        script = self.create_script(S3File(path=script))
+
+        reload_pipeline_object = self.create_pipeline_object(
+            object_class=ShellCommandActivity,
+            object_name=self.get_name("reload"),
+            input_node=[],
+            output_node=None,
+            resource=self.resource,
+            schedule=self.schedule,
+            script_uri=script,
+            script_arguments=script_arguments,
+            max_retries=self.max_retries,
+            depends_on=depends_on
+        )
+        return reload_pipeline_object
+
+
+    def create_and_load_redshift(self, table_definition, input_node):
+        table = self.get_table_from_def(table_definition)
+
+        if isinstance(input_node, dict):
+            input_paths = [i.path().uri for i in input_node.values()]
+        else:
+            input_paths = [input_node.path().uri]
+
+        script_arguments = [
+            '--table_definition=%s' % table.sql().sql(),
+            '--s3_input_paths'
+        ]
+        script_arguments.extend(input_paths)
+
+        steps_path = os.path.abspath(os.path.dirname(__file__))
+        script = os.path.join(steps_path, const.CREATE_LOAD_SCRIPT_PATH)
+        script = self.create_script(S3File(path=script))
+
+        # Create output_node based on output_path
+        base_output_node = self.create_s3_data_node()
+
+        create_and_load_pipeline_object = self.create_pipeline_object(
+            object_class=ShellCommandActivity,
+            object_name=self.get_name("create_and_load"),
+            input_node=[input_node],
+            output_node=base_output_node,
+            resource=self.resource,
+            schedule=self.schedule,
+            script_uri=script,
+            script_arguments=script_arguments,
+            max_retries=self.max_retries,
+            depends_on=self.depends_on
+        )
+        return create_and_load_pipeline_object
```

Comment:
should we move all these to lib so that the steps can share them with an import ?